README.md

```diff
@@ -102,9 +102,11 @@ uses the hostname.  kubelet allows this name to be overridden with
  -loglevel=4 \
  -k8s-apiserver="http://$CENTRAL_IP:8080" \
  -logfile="/var/log/openvswitch/ovnkube.log" \
- -init-master=$NODE_NAME -cluster-subnet="$CLUSTER_IP_SUBNET" \
```

Comment:
We do not have to run --init-node necessarily on the master node, is it?
Let this be run separately through the node initialization procedure.

Reply:
You mean run 2 ovnkube on master? Why?

Reply:
I have added a note below - that clarifies?